src/main/scala/org/scalacheck/Test.scala

```diff
@@ -11,6 +11,8 @@ package org.scalacheck
 
 import Prop.Arg
 
+import scala.collection.immutable.Seq
+
```

Comment:
This is because `Test.checkProperties` return type is defined as `Seq[(String,Result)]`.